docs/book/installation/installation.rst

```diff
@@ -29,7 +22,7 @@ for your custom e-commerce application.
 Initiating A New Sylius Project
 -------------------------------
 
-To create a new project using Sylius Standard Edition, run this command:
+Create your new project by running:
```

Comment:
Why? :)

Reply:
```suggestion
To begin creating your new project using Sylius Standard Edition, run this command:
```

Reply:
Let's say its one of the first info I read about Sylius. How can I know what is `Sylius Standard Edition` and whether I want to use it or whatever the alternative is?

Reply:
```suggestion
To begin creating your new project, run this command:
```